CHANGES.md

```diff
@@ -3,6 +3,8 @@ Changelog for singletons project
 
 2.5
 ---
+* Add `currySigma` and `uncurrySigma` to `Data.Singletons.Sigma`.
```

Comment:
Unfortunately, this didn't make the time window for the 2.5 release. Can you move this to a new `next` section?